hedera-node/hedera-app/src/main/java/com/hedera/node/app/Hedera.java

```diff
@@ -598,11 +601,7 @@ public void onStateInitialized(
         this.platform = requireNonNull(platform);
         if (state.getReadableStates(PlatformStateService.NAME).isEmpty()) {
             initializeStatesApi(
-                    state,
-                    metrics,
-                    trigger,
-                    RosterUtils.buildAddressBook(platform.getRoster()),
-                    platform.getContext().getConfiguration());
+                    state, metrics, trigger, null, platform.getContext().getConfiguration());
```

Comment:
I thought that this should not be genesis only since we don't explicitly set `trigger` to be genesis only, but given the `if` it might be the case that we end up here in genesis as well.
Initially I was using the roster from `rosterHistory` and converted it to addressbook here, but then updated it to `null` since that's how it is in https://github.com/hashgraph/hedera-services/pull/16843.

Reply:
As long as we can verify this doesn't result in an NPE, I'm fine with it